### PR TITLE
Just PR for virtualize UI to review  

### PR DESCRIPTION
**Pull request by xxxle0**
Closes #5290

Just a experimental PR for reviewing UI after change css

### CODE REVIEW

**Comment by tw15egan** [comment]
Looks good! Not seeing any issues with the Primary button or any of the filled variants, but I'm seeing some issues with the Tertiary button w/ Icon. It seems like when it receives focus, it shifts the icon to the left a bit. Here is an example:

**Tertiary**
tertiary

**Primary**, for reference
primary

**Comment by xxxle0** [comment]
Oh, nice catch 👍 let me investigate it

**Comment by xxxle0** [comment]
hi @tw15egan, Tertiary button seems working fine in my dev local. Just only 1 concern about the padding of tooltip will increase 1px after this change. 
Screen Shot 2020-02-07 at 11 25 18 AM

**Comment by tw15egan** [comment]
Hmm, I'm not seeing that issue with the tooltip button... 

Screen Shot 2020-02-07 at 11 25 04 AM

What browser/os are you using?

**Comment by xxxle0** [comment]
macOS Mojave 10.14.6 and Chrome 79.0.3945

**Review by emyarod — COMMENTED** [review]
it looks like tertiary button focus padding will need to be adjusted to account for the outline and border changes to prevent the icon from being shifted (related https://github.com/carbon-design-system/carbon/pull/5201/)

**Comment by emyarod** [comment]
this is related to https://github.com/carbon-design-system/carbon/issues/5263 right?

**Comment by laurenmrice** [comment]
when you have a chance could you rebuild so the react link doesn't fail. thanks !

**Comment by tw15egan** [comment]
Just triggered a rebuild, should be ready shortly

**Review by laurenmrice — CHANGES_REQUESTED** [review]
The focus border looks good and is 2px. 👍🏻

There still needs to be a 1px transparent gap, right now its 2px gap. 
Ex:
Artboard

**Comment by xxxle0** [comment]
hmm weird, this is what i see in my local  
Screen Shot 2020-02-18 at 9 31 31 PM

File: packages/components/src/globals/scss/_theme-tokens.scss [hunk 1]
```diff
@@ -131,12 +131,12 @@ $button-padding-ghost-sm: calc(0.375rem - 3px) 1rem !default;
...
 /// @access public
 /// @group button
-$button-border-width: 3px !default;
+$button-border-width: 2px !default;
 
 /// @type Number
 /// @access public
 /// @group button
-$button-outline-width: 1px !default;
+$button-outline-width: 2px !default;
```
Comment (tw15egan) [inline packages/components/src/globals/scss/_theme-tokens.scss:139]: I think we just need to keep this at `1px`

**Review by tw15egan — CHANGES_REQUESTED** [review]

File: packages/components/src/globals/scss/_theme-tokens.scss (hunk 1 above)
Comment (xxxle0) [reply packages/components/src/globals/scss/_theme-tokens.scss:139]: ok i got it

Comment (tw15egan) [reply packages/components/src/globals/scss/_theme-tokens.scss:139]: I originally read it as 2px on both as well 😃

**Review by laurenmrice — CHANGES_REQUESTED** [review]
The only thing left I am seeing if that the Tertiary button is still getting a 3px focus border. Everything else looks great !!!

Screen Shot 2020-02-18 at 9 45 23 AM

**Comment by xxxle0** [comment]
hi @tw15egan , @laurenmrice , can you guys review this PR again ?

File: packages/components/src/globals/scss/_theme-tokens.scss (hunk 1 above)
Comment (tw15egan) [reply packages/components/src/globals/scss/_theme-tokens.scss:139]: We need to keep this at `1px`

Comment (laurenmrice) [reply packages/components/src/globals/scss/_theme-tokens.scss:139]: yeah i think once that is fixed we should be good^

**Comment by xxxle0** [comment]
here is the outline 2px
Screen Shot 2020-02-26 at 8 40 07 AM
here is the outline 1px
Screen Shot 2020-02-26 at 8 41 19 AM

i think it should be 2px. Btw if we only change button-border-width, the last button outline will difference from another buttons

**Comment by tw15egan** [comment]
@xxxle0 Then we will need some sort of override on icon only buttons to make sure they are consistent with the rest of the buttons. Design spec is 1px so that is what we will adhere to.

**Comment by xxxle0** [comment]
yeah ok i got it. thanks @tw15egan

**Comment by tw15egan** [comment]
@xxxle0 do you have any plans to finish up this PR?

**Comment by tw15egan** [comment]
Seems like this issue will be bundled in with some larger button changes in https://github.com/carbon-design-system/carbon/pull/5618, so I'm going to close this out.

**Pull request closed without merging**